diff --git a/tasks/yard.rake b/tasks/yard.rake
index 75817cd..912d056 100644
--- a/tasks/yard.rake
+++ b/tasks/yard.rake
@@ -9,15 +9,4 @@ unless RUBY_PLATFORM == 'java' || RUBY_ENGINE == 'truffleruby'
   YARD::Rake::YardocTask.new
   CLEAN << '.yardoc'
   CLEAN << 'doc'
-
-  require 'yardstick/rake/verify'
-  Yardstick::Rake::Verify.new(:'yardstick:coverage') do |t|
-    t.threshold = 50
-    t.require_exact_threshold = false
-  end
-
-  desc 'Run yardstick to check yard docs'
-  task :yardstick do
-    sh "yardstick 'lib/**/*.rb'"
-  end
 end
