diff --git a/spec/spec_helper.rb b/spec/spec_helper.rb
index 02a4c45..4cf1109 100644
--- a/spec/spec_helper.rb
+++ b/spec/spec_helper.rb
@@ -51,15 +51,12 @@ end
 # SimpleCov configuration
 #
 require 'simplecov'
-require 'simplecov-lcov'
-require 'simplecov-rspec'
 
 def ci_build? = ENV.fetch('GITHUB_ACTIONS', 'false') == 'true'
 
 if ci_build?
   SimpleCov.formatters = [
     SimpleCov::Formatter::HTMLFormatter,
-    SimpleCov::Formatter::LcovFormatter
   ]
 end
 
@@ -272,11 +269,6 @@ end
 
 require 'rbconfig'
 
-SimpleCov::RSpec.start(list_uncovered_lines: ci_build?) do
-  # Avoid false positives in spec directory from JRuby, TruffleRuby, and Windows
-  add_filter '/spec/' unless mri? && windows?
-end
-
 # Make sure to require your project AFTER starting SimpleCov
 #
 require 'process_executer'
